packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -96,6 +100,7 @@ class _AddNewProductPageState extends State<AddNewProductPage>
   late final List<ProductFieldEditor> _editors;
   late final List<AnalyticsProductTracker> _trackers;
   final AddNewProductHelper _helper = AddNewProductHelper();
+  final PageController _pageController = PageController();
```

Comment:
Please ensure to dispose this controller

Reply:
Okay